src/averageAges.js

```diff
@@ -55,7 +81,19 @@ function calculateWomenAverageAge(people, withChildren) {
  * @return {number}
  */
 function calculateAverageAgeDiff(people, onlyWithSon) {
-  // write code here
+  const childArr = people.filter(el => {
```

Comment:
```suggestion
  const children = people.filter(el => {
```
Не треба писати Arr якщо змінна в множині названа, зрозуміло що це масив